app/Controllers/indexController.php

```diff
@@ -48,6 +48,8 @@ public function normalAction() {
 		}
 		Minz_View::prependTitle($title . ' · ');
 
+		FreshRSS_Context::$id_max = time() . '000000';
```

Comment:
Simplified logic relying solely on current time instead of the actual first element in DB. Should not matter because inserted elements after this request will have a newer ID.